cluster/saltbase/salt/kube-addons/kube-addons.sh

```diff
@@ -145,7 +145,7 @@ fi
 token_found=""
 while [ -z "${token_found}" ]; do
   sleep .5
-  token_found=$(${KUBECTL} get serviceaccount default -o template -t "{{with index .secrets 0}}{{.name}}{{end}}" || true)
+  token_found=$(${KUBECTL} get serviceaccount kube-system -o template -t "{{with index .secrets 0}}{{.name}}{{end}}" || true)
```

Comment:
i think this should be:
```
${KUBECTL} --namespace=kube-system get serviceaccount default...
```
the default service account is called default in all namespaces.